client/cmd/format.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"alda.io/client/color"
+	"alda.io/client/help"
+	log "alda.io/client/logging"
+	"alda.io/client/parser"
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+var formatInputFile string
+var formatOutputFile string
+
+func init() {
+	formatCmd.Flags().StringVarP(
+		&formatInputFile, "file", "f", "", "Format an Alda file",
+	)
+
+	formatCmd.Flags().StringVarP(
+		&formatOutputFile, "output", "o", "", "A separate file to output formatted Alda to",
+	)
+}
+
+var formatCmd = &cobra.Command{
+	Use: "format",
+	Hidden: true,
+	Short: "Format Alda source code",
+	Long: `Format Alda source code
+
+---
+
+Source code must be provided by specifying the path to a file (-f, --file):
+  alda format -f path/to/my-score.alda
+
+In this case, the score will be formatted in-place (i.e. overwritten).
```

Comment:
To make this formatter more flexible, I think it should print the result to stdout by default, and we should include a flag to enable overwriting the file.

`sed` and [`gofmt`](https://pkg.go.dev/cmd/gofmt) both operate this way -- they write the result to stdout by default, and there is a flag (`sed -i`, `gofmt -w`) to opt into overwriting the file.

Reply:
Good suggestion. Done. Now `-f` specifies input file, default printing to `os.Stdout`. Specifying `-o` will overwrite the input file.